hive-interface.js

```diff
@@ -214,13 +216,21 @@ class Hive {
 
 	// Left for backwards compatibility
 	async customJsonNoQueue(id, json, account, key, use_active) {
+		if(this.accounts_used.has(account)) {
+			let count = this.accounts_used.get(account);
+			if(count => 4) 
+				// If we are at the limit, queue instead
+				return this.custom_json(id, json, account, key, use_active);
+			else
```

Comment:
I think you need to move this down or you will never set the account.

Reply:
Ah, yea, need to add the line handling new accounts at the bottom of that code.